operation/fhir-operation-reindex/src/main/java/com/ibm/fhir/operation/reindex/ReindexOperation.java

```diff
@@ -137,6 +139,14 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                             }
                             resourceCount = val;
                         }
+                    } else if (PARAM_FORCE.equals(parameter.getName().getValue())) {
```

Comment:
we could store parameter.getName().getValue() to a local variable above and same some characters in all these else if lines...

Reply:
fixed